packages/python-ccrawl/PKGBUILD

```diff
@@ -0,0 +1,37 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+pkgbase=python-ccrawl
+pkgname=('python-ccrawl')
+_pkgname=ccrawl
+pkgver=1.2
+pkgrel=1
+pkgdesc='Ccrawl uses clang to build a database related to various C/C++ data structures.'
+arch=('any')
+groups=('blackarch' 'blackarch-binary' 'blackarch-forensic')
+url='https://github.com/bdcht/ccrawl'
+license=('GPL3')
+makedepends=('git' 'python-pip')
+source=("git+https://github.com/bdcht/$_pkgname.git")
+sha512sums=('SKIP')
+
+prepare() {
+  cp -a "$_pkgname"{,-2}
```

Comment:
remove `{,-2}` since you don't package the py2 version

Reply:
Also remove double quote when there is no no dashes or slashes

Reply:
This should be removed with no py2 deps ? If so; will resolve. Thanks.